test/JDBC/expected/BABEL-4869-vu-verify.out

```diff
@@ -0,0 +1,253 @@
+-- Basic Case: No parameters
+EXEC sp_describe_undeclared_parameters 'SELECT * FROM babel_4869_vu_prepare_t1'
+GO
+~~START~~
+int#!#varchar#!#int#!#nvarchar#!#smallint#!#tinyint#!#tinyint#!#int#!#varchar#!#varchar#!#varchar#!#nvarchar#!#int#!#varchar#!#varchar#!#varchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#varchar#!#int#!#int
+~~END~~
+
+
+-- Single Parameter Case: WHERE clause with a single parameter
+EXEC sp_describe_undeclared_parameters 'SELECT * FROM babel_4869_vu_prepare_t1 WHERE Column1 = @Param1'
+GO
+~~START~~
+int#!#varchar#!#int#!#nvarchar#!#smallint#!#tinyint#!#tinyint#!#int#!#varchar#!#varchar#!#varchar#!#nvarchar#!#int#!#varchar#!#varchar#!#varchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#varchar#!#int#!#int
+~~END~~
+
+
+-- Multiple Parameters Case: WHERE clause with multiple parameters
+EXEC sp_describe_undeclared_parameters 'SELECT * FROM babel_4869_vu_prepare_t1 WHERE Column1 = @Param1 AND Column2 = @Param2'
+GO
+~~START~~
+int#!#varchar#!#int#!#nvarchar#!#smallint#!#tinyint#!#tinyint#!#int#!#varchar#!#varchar#!#varchar#!#nvarchar#!#int#!#varchar#!#varchar#!#varchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#varchar#!#int#!#int
+~~END~~
+
+
+-- Nested Queries Case: Subquery in WHERE clause
+EXEC sp_describe_undeclared_parameters 'SELECT * FROM babel_4869_vu_prepare_t1 WHERE Column1 IN (SELECT Column3 FROM babel_4869_vu_prepare_t2 WHERE Column2 = @Param)'
+GO
+~~START~~
+int#!#varchar#!#int#!#nvarchar#!#smallint#!#tinyint#!#tinyint#!#int#!#varchar#!#varchar#!#varchar#!#nvarchar#!#int#!#varchar#!#varchar#!#varchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#varchar#!#int#!#int
+~~END~~
+
+
+-- Join Conditions Case: JOIN with a parameter in the condition
+EXEC sp_describe_undeclared_parameters 'SELECT * FROM babel_4869_vu_prepare_t1 JOIN babel_4869_vu_prepare_t2 ON babel_4869_vu_prepare_t1.Column2 = babel_4869_vu_prepare_t2.Column2 WHERE babel_4869_vu_prepare_t1.Column1 = @Param'
+GO
+~~START~~
+int#!#varchar#!#int#!#nvarchar#!#smallint#!#tinyint#!#tinyint#!#int#!#varchar#!#varchar#!#varchar#!#nvarchar#!#int#!#varchar#!#varchar#!#varchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#varchar#!#int#!#int
+~~END~~
+
+
+-- Error Handling Case: Syntax error
```

Comment:
Let's also add few happy scenarios at the end in which we correctly call this procedure and get correct result. This will also help in verifying if we are leaking the value of static variable we introduced.

Reply:
Few scenarios which returns rows are already added. Please check line 62, 73, 155